code/game/objects/items/peppercloud.dm

```diff
@@ -0,0 +1,104 @@
+//pepperspray
+/obj/item/reagent_containers/peppercloud_deployer
+	name = "pepper-cloud deployer"
+	desc = "Manufactured by UhangInc, upon activation a pepper-cloud will be deployed slowing down and disorienting anyone who enters it."
+	icon = 'icons/obj/items_and_weapons.dmi'
+	icon_state = "pepperspray"
+	item_state = "pepperspray"
+	lefthand_file = 'icons/mob/inhands/equipment/security_lefthand.dmi'
+	righthand_file = 'icons/mob/inhands/equipment/security_righthand.dmi'
+	volume = 50
+	list_reagents = list(/datum/reagent/consumable/condensedcapsaicin = 50)
+
+	item_flags = NOBLUDGEON | ISWEAPON
+	reagent_flags = OPENCONTAINER
+	slot_flags = ITEM_SLOT_BELT
+	throwforce = 0
+	w_class = WEIGHT_CLASS_SMALL
+	throw_speed = 3
+	throw_range = 7
+
+	var/cooldown_time = 0
+	var/activation_cooldown = 30 SECONDS
```

Comment:
I don't believe the cooldown is necessary, given it can only have two sprays. If a cooldown between sprays is 100% necessary for balance, it should simply disperse all of its contents in one spray, requiring a refill before being used again.

Arbitrary cooldowns just feel wrong - what is actually preventing the spray from being used again immediately?

Reply:
Made the cooldown 2 seconds instead